vllm/model_executor/layers/quantization/utils/fp8_utils.py

```diff
@@ -0,0 +1,116 @@
+from typing import List, Optional, Tuple, Union
+
+import torch
+
+from vllm import _custom_ops as ops
+from vllm.model_executor.utils import set_weight_attrs
+from vllm.platforms import current_platform
+
+
+def cutlass_fp8_supported() -> bool:
+    capability = current_platform.get_device_capability()
+    capability = capability[0] * 10 + capability[1]
+
+    return ops.cutlass_scaled_mm_supports_fp8(capability)
+
+
+def per_tensor_quantize(tensor: torch.Tensor,
+                        inv_scale: Union[float, torch.Tensor]) -> torch.Tensor:
+    finfo = torch.finfo(torch.float8_e4m3fn)
+    qweight = (tensor / inv_scale).clamp(min=finfo.min, max=finfo.max)
+    return qweight.to(torch.float8_e4m3fn)
```

Comment:
Actually we should be able to use `ops.scaled_fp8_quant` to replace this function?